packages/liferay-npm-scripts/src/jsp/dedent.js

```diff
@@ -85,9 +85,7 @@ function dedent(input, tabWidth = 4) {
 		dedented.pop();
 	}
 
-	const lastMinimum = isFinite(minimum) ? Math.floor(minimum / tabWidth) : 0;
-
-	return [dedented.join('\n'), lastMinimum];
```

Comment:
So this was the problem. Previously we just detected the indent of the script contents and trusted that it was ok.

The new approach is to just skip the detection and let the opening `<script>` or `<aui:script>` tag call the shots.